protos/sigstore_common.proto

```diff
@@ -69,6 +69,9 @@ enum PublicKeyDetails {
         PKIX_RSA_PKCS1_2048_SHA256 = 9;
         PKIX_RSA_PKCS1_3072_SHA256 = 10;
         PKIX_RSA_PKCS1_4096_SHA256 = 11;
+        RSA_PSS_2048_SHA256 = 16;
+        RSA_PSS_3072_SHA256 = 17;
+        RSA_PSS_4096_SHA256 = 18;
```

Comment:
From https://github.com/sigstore/protobuf-specs/pull/212#discussion_r1476477104: maybe we want to explicitly prefix these with `PKIX_`, emphasizing that they're encoding according to RFC 4055? Thoughts?